portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var curSlide = 0; //by default, display first slide
+const totalSlides = 3; //total number of slides
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/*
+ * Advance forward by one slide
+ */
+function forward() {
+  if (curSlide < totalSlides - 1) {
+    updateSlide(curSlide + 1);
+  } else { //curSlide = totalSlides
```

Comment:
nit: Even though this is just a comment and not executed, I think it's worth changing the `=` to `==` or `===` for clarity.